Source/Scene/Model.js

```diff
@@ -4076,7 +4076,7 @@ define([
             context : context,
             typedArray : typedArray,
             usage : BufferUsage.STATIC_DRAW,
-            indexDatatype : IndexDatatype.UNSIGNED_INT
+            indexDatatype : IndexDatatype.UNSIGNED_SHORT
```

Comment:
We should test a model that has more than 65536 vertices. It will likely fail if `UNSIGNED_SHORT` is hardcoded. Could the index datatype be determined when reading then number of vertices in the compressed data?

Reply:
Fixed this so it uses `UNSIGNED_INT` for more vertices. Do you know where I could find a model to test?